recipe_nonWF_early.trees.pedigree

```diff
@@ -0,0 +1,229 @@
+generation	stage	individual	age	parent1	parent2
```

Comment:
let's not commit these temporary files produced by tests!

Reply:
Oops. Well spotted. Must have been left from a previous trial. Sorry!